person.rb

```diff
@@ -1,3 +1,5 @@
+require './corrector'
+
```

Comment:
- [ ] In the Person class initialize method, an instance of Corrector should be set as stated in the requirements. Kindly add an instance of the Corrector class, which should be something like this `@corrector = Corrector.new()`